src/test/java/org/owasp/validator/html/test/AntiSamyTest.java

```diff
@@ -2576,4 +2576,4 @@ public void testMalformedPIScan() {
       fail("Parser should not throw a non-ScanException");
     }
   }
-}
+}
```

Comment:
I prefer to have all files end with a newline. I think there were two files changed marked with a missing newline at the end. Can you fix those two? The other was AntiSamyRules.xml. In fact, if you undo the change in that file I think there are no other changes.